### ISSUE TITLE
Config isn't loaded with mocha

### ISSUE THREAD

**Issue by GCorbel** [samselikoff/ember-cli-mirage#1396]
I have this simple test : 

```
import { describe, it, beforeEach, afterEach } from 'mocha';
import startApp from 'form/tests/helpers/start-app';
import destroyApp from 'form/tests/helpers/destroy-app';
import { setupApplicationTest } from 'ember-mocha';
import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';

describe('Acceptance | subscription', function() {
  setupApplicationTest();
  setupMirage();
  let application;

  beforeEach(function() {
    application = startApp();
  });

  afterEach(function() {
    destroyApp(application);
  });

  it('can fill a form', async function() {
    await visit('/subscription');
  });
});
```

And I have this config file : 

```
export default function() {
  this.get('/forms');
}
```

When I run the test I have `Your Ember app tried to GET '/forms' ...`.

The solution I found is to add `import mirageConfig from 'form/mirage/config';` and do `this.server.loadConfig(mirageConfig); ` in the beforeEach hook.

Does this should be done with `setupMirage()` ?

**Comment by samselikoff** [comment]
hm, looks like the mirage config isn't being picked up by default. You should not have to do that manually. Where is your mirage config?

**Comment by GCorbel** [comment]
My mirage config is under `mirage/config.js`, the default path.

**Comment by samselikoff** [comment]
Seems weird. Any chance you could post a minimal reproduction?

**Comment by GCorbel** [comment]
I will try to reproduce the issue with a new app but the app I'm working on isn't very complicated. I keep you up to date.

**Comment by samselikoff** [comment]
I believe ember-mocha's README addresses this [here](https://github.com/emberjs/ember-mocha#setup-tests). I think you need to do something like this:

```js
const hooks = setupApplicationTest();
setupMirage(hooks);
```

Closing for now but please ping here if you're still having trouble!

**Comment by kennydp** [comment]
I have the same issue. I started using mocha in our project and the config can't be found when testing. It does load when I just serve with mirage enabled. I tried checking how this was done and the `resolveRegistration` function just returns undefined (for either `base-config` or `test-config`)

I have used the steps that are in the ember-mocha's readme, but this did not seem to load the config.
Our setup may be a bit more complex, but I'll try to see if I can reproduce it in a new app and try to get back to you this week.
